arch/RISCV/RISCVDisassembler.c

```diff
@@ -0,0 +1,417 @@
+//===-- RISCVDisassembler.cpp - Disassembler for RISCV --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+/* Capstone Disassembly Engine */
+/* By Rodrigo Cortes Porto <porto703@gmail.com>, 2018 */
+
+#ifdef CAPSTONE_HAS_RISCV
+
+#include <stdio.h>		// DEBUG
+#include <stdlib.h>
+#include <string.h>
+
+#include "../../cs_priv.h"
+#include "../../utils.h"
+
+#include "../../MCInst.h"
+#include "../../MCInstrDesc.h"
+#include "../../MCFixedLenDisassembler.h"
+#include "../../MCRegisterInfo.h"
+#include "../../MCDisassembler.h"
+#include "../../MathExtras.h"
+#include "RISCVBaseInfo.h"
+#include "RISCVDisassembler.h"
+
+///
+static uint64_t getFeatureBits(int mode) 
+{
+  	// support everything
+  	return (uint64_t) - 1;
```

Comment:
This is in combination of [checkDecoderPredicate](https://github.com/aquynh/capstone/pull/1401/files#diff-e16c5f26d5ad44a6c9fa0a1f9ea7a007R1071) basically equivalent to target RV64IMAFDC.
`mode` could be evaluated here, imho

Reply:
how to explain all the arch `getFeatureBits(mode)` return -1 ?

Reply:
Sorry, can you please explain your question?

Reply:
Thanks for your concerns, After digging into RISCV specification and LLVM RISCV port again.
looks like need adds a new CS_MODE_RISCV64.
Already done, Please check again.